src/autoload/Pyegi.lua

```diff
@@ -387,8 +410,10 @@ local function post_init(sub, sel)
 				sub.insert(filtered_desired_lines[line_number]+1, l2)
 			end
 			APT("")
+		elseif btn2 == "Back" then
```

Comment:
I couldn't add comment for other btns2 references... btw, they are related to 👇 comment